### PR TITLE
Add disclaimer to code files and readmes

### PR DESCRIPTION
**Pull request by johnathanDOS**
Added the following disclaimer copy wherever there was a license. We should add this disclaimer to any new READMEs and code files that we add to this repository, and if this PR misses any, then we should update those, too.

> 
> **WARRANTY DISCLAIMER.**
> 
> * General. TO THE MAXIMUM EXTENT PERMITTED BY APPLICABLE LAW, MISTY ROBOTICS PROVIDES THIS SAMPLE SOFTWARE “AS-IS” AND DISCLAIMS ALL WARRANTIES AND CONDITIONS, WHETHER EXPRESS, IMPLIED, OR STATUTORY, INCLUDING THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, TITLE, QUIET ENJOYMENT, ACCURACY, AND NON-INFRINGEMENT OF THIRD-PARTY RIGHTS. MISTY ROBOTICS DOES NOT GUARANTEE ANY SPECIFIC RESULTS FROM THE USE OF THIS SAMPLE SOFTWARE. MISTY ROBOTICS MAKES NO WARRANTY THAT THIS SAMPLE SOFTWARE WILL BE UNINTERRUPTED, FREE OF VIRUSES OR OTHER HARMFUL CODE, TIMELY, SECURE, OR ERROR-FREE.
> * Use at Your Own Risk. YOU USE THIS SAMPLE SOFTWARE AND THE PRODUCT AT YOUR OWN DISCRETION AND RISK. YOU WILL BE SOLELY RESPONSIBLE FOR (AND MISTY ROBOTICS DISCLAIMS) ANY AND ALL LOSS, LIABILITY, OR DAMAGES, INCLUDING TO ANY HOME, PERSONAL ITEMS, PRODUCT, OTHER PERIPHERALS CONNECTED TO THE PRODUCT, COMPUTER, AND MOBILE DEVICE, RESULTING FROM YOUR USE OF THIS SAMPLE SOFTWARE OR PRODUCT.
> 
> Please refer to the Misty Robotics End User License Agreement for further information and full details: https://www.mistyrobotics.com/legal/end-user-license-agreement/

### CODE REVIEW

File: C#/Sample Projects/Navigation/SharedTypes/ChargerDock.cs [hunk 1]
```diff
@@ -11,7 +11,29 @@
...
+* 	 YOUR OWN DISCRETION AND RISK. YOU WILL BE SOLELY RESPONSIBLE FOR (AND MISTY
+* 	 ROBOTICS DISCLAIMS) ANY AND ALL LOSS, LIABILITY, OR DAMAGES, INCLUDING TO
+* 	 ANY HOME, PERSONAL ITEMS, PRODUCT, OTHER PERIPHERALS CONNECTED TO THE PRODUCT,
+* 	 COMPUTER, AND MOBILE DEVICE, RESULTING FROM YOUR USE OF THIS SAMPLE SOFTWARE
+* 	 OR PRODUCT.
+* 
+* 	 Please refer to the Misty Robotics End User License Agreement for further
+* 	 information and full details:
+* 	 	https://www.mistyrobotics.com/legal/end-user-license-agreement/
+******************************************************************************/–
```
Comment (adam-misty) [inline C#/Sample Projects/Navigation/SharedTypes/ChargerDock.cs]: Computers are really picky about stray characters.  "-" at the end.

File: C#/Libraries/SkillTools/README.md [hunk 2]
```diff
@@ -3,6 +3,17 @@
...
+* Use at Your Own Risk. YOU USE THIS SAMPLE SOFTWARE AND THE PRODUCT AT YOUR OWN DISCRETION AND RISK. YOU WILL BE SOLELY RESPONSIBLE FOR (AND MISTY ROBOTICS DISCLAIMS) ANY AND ALL LOSS, LIABILITY, OR DAMAGES, INCLUDING TO ANY HOME, PERSONAL ITEMS, PRODUCT, OTHER PERIPHERALS CONNECTED TO THE PRODUCT, COMPUTER, AND MOBILE DEVICE, RESULTING FROM YOUR USE OF THIS SAMPLE SOFTWARE OR PRODUCT.
+
+Please refer to the Misty Robotics End User License Agreement for further information and full details: https://www.mistyrobotics.com/legal/end-user-license-agreement/
+
+--- 
+
+*Copyright 2020 Misty Robotics*<br>
```
Comment (adam-misty) [inline C#/Libraries/SkillTools/README.md:17]: Should these <br>'s be in the README.md files? They disappear when I view the file in this PR tool, but when I open them with NOTEPAD++ or Chrome on my PC I see the text "<br>"

Comment (johnathanDOS) [reply C#/Libraries/SkillTools/README.md:17]: Do you mean the disclaimer copy in general? Yes, that should be in the README.md files  👍

File: C#/Sample Projects/Navigation/SharedTypes/ChargerDock.cs (hunk 1 above)
Comment (johnathanDOS) [reply C#/Sample Projects/Navigation/SharedTypes/ChargerDock.cs]: Shoot. Thanks! Fixed.

**Review by adam-misty — APPROVED** [review]

**Pull request merged**